app/src/main/java/com/github/libretube/adapters/CommentsAdapter.kt

```diff
@@ -9,18 +9,18 @@ import androidx.recyclerview.widget.RecyclerView
 import com.github.libretube.R
 import com.github.libretube.formatShort
 import com.github.libretube.obj.Comment
-import com.github.libretube.obj.StreamItem
 import com.squareup.picasso.Picasso
 
 class CommentsAdapter(private val comments: MutableList<Comment>):  RecyclerView.Adapter<ViewHolder>(){
 
     fun updateItems(newItems: List<Comment>){
         comments.addAll(newItems)
-        notifyItemRangeInserted(comments.size, newItems.size)
+        var commentsSize = comments.size
```

Comment:
```kotlin
var commentsSize = comments.size
```
should be before:
```kotlin
comments.addAll(newItems)
```

Reply:
why can't you? hmm

Reply:
I think he has this disabled:
![image](https://user-images.githubusercontent.com/20838718/167537119-d876b268-6005-435d-b0c6-645ce6b914a3.png)

Reply:
I have it enabled